media/css/pebbles/components/_global-nav.scss

```diff
@@ -0,0 +1,594 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Common variables
+
+$color-firefox-light-orange: #ff9500;
+
+$mq-nav-tablet:     'screen and (min-width: 820px)';
+$mq-nav-desktop:    'screen and (min-width: 1050px)';
+
+/* -------------------------------------------------------------------------- */
+// Horizontal top navigation
+
+.moz-global-nav {
+    position: relative;
+    background: #fff;
+    overflow: hidden;
+
+    a:link,
+    a:visited {
+        color: #000;
+        font-weight: bold;
+        text-decoration: none;
+    }
+
+    a:hover,
+    a:focus,
+    a:active {
+        color: #000;
+        text-decoration: underline;
+        transition: color 0.1s ease-in-out;
+    }
+
+    // Three-dotted nenu button
+    .nav-button-menu {
```

Comment:
So I'm generally siding these days that buttons [shouldn't get a hand cursor](https://medium.com/simple-human/buttons-shouldnt-have-a-hand-cursor-b11e99ca374b) in general. Here I agree it's not immediately obvious that this is a button however, so I think an extra visual cue could be worthwhile.